### PR TITLE
BINF-481: implement to prod

### PR DESCRIPTION
**Pull request by Shenglai**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM quay.io/ncigdc/gatk:3.7 AS gatk
-FROM python:3.7-slim AS python
+FROM quay.io/ncigdc/python37 AS python
 FROM openjdk:slim
```
Comment (Shenglai) [inline Dockerfile:3]: not sure if we have ours that could avoid the limits

File: bin/mutect2_tool [hunk 2]
```diff
@@ -5,5 +5,5 @@ set -eo pipefail
 case "$1" in
 	test) python -m pytest tests;;
 	*version) python -m mutect2_tool --version;;
```
Comment (Shenglai) [inline bin/mutect2_tool:7]: i dont think this version is working.

File: Dockerfile [hunk 3]
```diff
@@ -1,6 +1,6 @@
 FROM quay.io/ncigdc/gatk:3.7 AS gatk
-FROM python:3.7-slim AS python
-FROM openjdk:slim
+FROM quay.io/ncigdc/python37 AS python
+FROM openjdk:8u282-slim
```
[the remark this answers is not in the document]
Comment (Shenglai) [reply Dockerfile:3]: Also discover that the script isn't catching the exit code correctly for individual process. CWL returns complete when the mutect2 actually failed due to the wrong jdk environment.

Gonna take a look at the implementation tmr.

[the remark this answers is not in the document]
Comment (czyszCTDS) [reply Dockerfile:3]: I will copy this openjdk tag to quay so it doesn't trigger any `toomanypulls` errors

**Pull request merged**